Mathlib/Analysis/Normed/Lp/ProdLp.lean

```diff
@@ -449,6 +449,12 @@ lemma prod_continuous_toLp : Continuous (@toLp p (α × β)) := continuous_id
 @[continuity, fun_prop]
 lemma prod_continuous_ofLp : Continuous (@ofLp p (α × β)) := continuous_id
 
+/-- `WithLp.equiv` as a homeomorphism. -/
+def homeomorph_prod : α × β ≃ₜ WithLp p (α × β) where
```

Comment:
I'm guessing we want this one and the next change to be the other way round, to match `WithLp.equiv`?
Let's also make sure to add the `.toEquiv = WithLp.equiv` lemmas